runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -109,6 +109,7 @@ public AddressSpaceView(@Nonnull final CorfuRuntime runtime) {
         if (maxCacheEntries == 0) {
             log.warn("Since AddressSpaceView readCache size is 0, " +
                     "overriding CorfuRuntime bulkReadSize and checkpointReadBatchSize to 1.");
+            runtime.getParameters().setBulkReadSize(1);
```

Comment:
Also setStreamBatchSize(1)?

If streamBatchSize is 10 (default), it might not help with reducing the num of RPCs even if we set bulkReadSize as 1.
Because on a getBatch(), it decides #addresses based on streamBatchSize https://github.com/CorfuDB/CorfuDB/blob/master/runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java#L426

Reply:
Good catch. Just adjusting streamBatchSize to 1 is sufficient. I'll close this PR and create a new one.